internal/scraper/scraper.go

```diff
@@ -804,16 +805,25 @@ func httpSettingsToBBEModule(ctx context.Context, logger zerolog.Logger, setting
 
 	m.HTTP.NoFollowRedirects = settings.NoFollowRedirects
 
-	if len(settings.Headers) > 0 {
-		m.HTTP.Headers = make(map[string]string)
+	userAgentHeader := "User-Agent"
+
+	m.HTTP.Headers = map[string]string{
+		userAgentHeader: version.UserAgent(),
 	}
+
 	for _, header := range settings.Headers {
 		parts := strings.SplitN(header, ":", 2)
+
 		var value string
 		if len(parts) == 2 {
 			value = strings.TrimLeft(parts[1], " ")
 		}
-		m.HTTP.Headers[parts[0]] = value
+
+		if strings.ToTitle(parts[0]) == userAgentHeader {
```

Comment:
bit confused about what is going on here, can we add a comment about it?

Reply:
Headers are case insensitive (https://tools.ietf.org/html/rfc7230#section-3.2) so we need to fold them. `strings.EqualFold` folds both strings all the time, so it's a little more expensive, so it's better to have one string in the correct case and just fold the other one. I'll change this to use lowercase and maybe that's less obscure.